src/components/Dialog/VoiceLibraryPolicyDialog.vue

```diff
@@ -0,0 +1,170 @@
+              <BaseDocumentView>
+                <!-- eslint-disable-next-line vue/no-v-html -->
+                <div v-html="info.renderedPolicy"></div>
+              </BaseDocumentView>
+            </div>
+          </div>
+        </div>
+      </QCardSection>
+
+      <QSeparator />
+
+      <QCardActions>
+        <QSpace />
+        <QBtn
+          padding="xs md"
+          label="キャンセル"
+          unelevated
+          color="surface"
+          textColor="display"
+          class="q-mt-sm text-bold"
+          @click="handleCancel"
+        />
+        <QBtn
+          padding="xs md"
+          label="確認して続行"
+          unelevated
+          color="surface"
+          textColor="display"
+          class="q-mt-sm text-bold"
+          @click="handleOk"
+        />
+      </QCardActions>
+    </QCard>
+  </QDialog>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import { useDialogPluginComponent } from "quasar";
+import { SpeakerId } from "@/type/preload";
+import BaseDocumentView from "@/components/Base/BaseDocumentView.vue";
+import { useMarkdownIt } from "@/plugins/markdownItPlugin";
+
+const modelValue = defineModel<boolean>({ default: false });
+
+type CharacterPolicyInfo = {
+  id: SpeakerId;
+  name: string;
+  policy: string;
+  portraitPath: string;
+};
+
+const props = defineProps<{
+  characterPolicyInfos: CharacterPolicyInfo[];
+}>();
+
+defineEmits({
+  ...useDialogPluginComponent.emitsObject,
+});
+
+const { dialogRef, onDialogOK, onDialogCancel, onDialogHide } =
+  useDialogPluginComponent();
+
+const md = useMarkdownIt();
+
+const enrichedCharacterInfos = computed(() =>
+  props.characterPolicyInfos.map((info) => ({
+    id: info.id,
+    name: info.name,
+    portraitPath: info.portraitPath,
+    renderedPolicy: md.render(info.policy),
+  })),
+);
+
+const characterIds = computed(() =>
+  enrichedCharacterInfos.value.map((c) => c.id),
+);
+
+const handleCancel = () => {
+  onDialogCancel();
+};
+
+const handleOk = () => {
+  onDialogOK(characterIds.value);
+};
+
+const handleOpenUpdate = (isOpen: boolean) => {
+  if (!isOpen) {
+    onDialogHide();
+  }
+};
+</script>
+
+<style scoped lang="scss">
+@use "@/styles/v2/variables" as vars;
+@use "@/styles/v2/colors" as colors;
+@use "@/styles/v2/mixin" as mixin;
+
+.policy-dialog {
+  width: 700px;
+  max-width: 80vw;
+}
+
+.scrollable-area {
+  overflow-y: auto;
+  max-height: calc(100vh - 100px - 295px);
+}
+
+.character-policies {
+  display: flex;
+  flex-direction: column;
+  gap: vars.$gap-2;
+}
+
+.character-policy-item {
+  display: flex;
+  gap: vars.$gap-2;
+  padding: vars.$padding-2;
+  border: 1px solid colors.$border;
+  background-color: colors.$surface;
+  border-radius: vars.$radius-2;
+}
+
+.character-portrait-section {
+  width: 6rem;
+  height: 9rem;
+  display: flex;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.character-info-section {
+  flex: 1;
+  min-width: 0;
+}
```

Comment:
極端に長いURLの場合（ちび式じいの利用規約リンクが該当）ダイアログ最大幅でも文字がはみ出てしまっていました。
<img width="818" height="477" alt="image" src="https://github.com/user-attachments/assets/961022e5-d5a5-4603-8e8f-ce9088892f39" />

横幅が狭いときのみであれば現状許容できますが、最大幅で発生するとなると折り返すように変更したほうが良さそうです。
```suggestion
.character-info-section {
  flex: 1;
  min-width: 0;
  overflow-wrap: anywhere;
}
```